lib/Transforms/YosysOptimizer/LUTImporterTest.cpp

```diff
@@ -0,0 +1,164 @@
+#include "gmock/gmock.h"  // from @googletest
+#include "gtest/gtest.h"  // from @googletest
+#include "include/Dialect/Comb/IR/CombOps.h"
+#include "kernel/yosys.h"  // from @at_clifford_yosys
+#include "lib/Transforms/YosysOptimizer/LUTImporter.h"
+#include "mlir/include/mlir/Dialect/Arith/IR/Arith.h"  // from @llvm-project
+#include "mlir/include/mlir/IR/MLIRContext.h"          // from @llvm-project
+
+namespace mlir::heir {
+namespace {
+
+using ::testing::ElementsAreArray;
+using ::testing::Test;
+
+// Returns a list of cell names that are topologically ordered using the Yosys
+// toder output. This is extracted from the lines containing cells in the
+// output:
+// -- Running command `torder -stop * P*;' --
+
+// 14. Executing TORDER pass (print cells in topological order).
+// module test_add
+//   cell $abc$167$auto$blifparse.cc:525:parse_blif$168
```

Comment:
You don't end up using these names for anything besides equality comparison, do you? Well just in case, if you end up needing to do additional random string processing on them, you can also consider running `rename -enumerate`

https://yosyshq.readthedocs.io/projects/yosys/en/latest/cmd/rename.html

Which will automatically rename all the cells to have simpler increasing integer names.